lib/smart_answer/calculators/arrested_abroad.rb

```diff
@@ -1,14 +1,12 @@
 module SmartAnswer::Calculators
   class ArrestedAbroad
     # created for the help-if-you-are-arrested-abroad calculator
```

Comment:
Is this comment still relevant?

Reply:
Maybe not - but removing it didn't seem in scope for this PR.